pkg/apiserver/resthandler.go

```diff
@@ -234,7 +256,7 @@ func ListResource(r rest.Lister, rw rest.Watcher, scope RequestScope, forceWatch
 		opts := *out.(*api.ListOptions)
```

Comment:
In other words, this is mixing apiserver infrastructure with our own particular api.